src/data/local/github/tracks-node/track-3/steps/step-2.md

```diff
@@ -0,0 +1,16 @@
+# Step 1 - Integrating with the database
+
+### Estimated time: 4 hours
+
+Now you're going to fully integrate the Login mutation with the database. Since we don't have a mutation to create users (yet), we should create one directly on the database, so we have the conditions to test it properly.
+
+**NOTE:** One of the required data to create a user is the password. You should have read or heard already that storing users password in plain text on database is a bad ideia. If not, you're reading now: it **is** a bad ideia 🤦‍. The minimum security we should have on this case is to use a hash algorithm to store the password. This is not 100% secure method, but it helps already. Interested in this security matter? Check [this post](https://itnext.io/how-not-to-store-passwords-4955569e6e84) for additional details. Since this is only an onboard server, you don't need to have a ultra-master-blaster security method. Our main goal here is to show you the degrees of secutrity, and prevent you from beginning your projects with an ultra-master-blaster insecure method.
```

Comment:
```suggestion
**NOTE:** One of the required data to create a user is the password. You should have read or heard already that storing users password as plain text in the database is a bad idea. If not, you're reading now: it **is** a bad idea 🤦‍. The minimum security we should have on this case is to use a hash algorithm to store the password. This is not a 100% secure method, but it helps already. Interested in this security matter? Check [this post](https://itnext.io/how-not-to-store-passwords-4955569e6e84) for additional details. Since this is only an onboard server, you don't need to have a ultra-master-blaster security method. Our main goal here is to show you some levels of security and prevent you from beginning your projects with an ultra-master-blaster (we like that word) insecure method.
```